flexmeasures/data/models/planning/storage.py

```diff
@@ -92,7 +95,12 @@ def _prepare(self, skip_validation: bool = False) -> tuple:  # noqa: C901
         inflexible_device_sensors = self.flex_context.get(
             "inflexible_device_sensors", []
         )
-
+        soc_sensor = None
+        if isinstance(soc, Sensor):
+            soc_sensor = soc
+            soc_at_start = get_soc_sensor_value(soc, start)
+        elif (isinstance(soc, float) or isinstance(soc, int)) and soc > 0:
+            soc_at_start = soc
```

Comment:
Can be commented out to limit the scope of this PR to saving only (rather than saving and fetching).